content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -109,10 +114,11 @@ Create and send an invoice for prepaid services
 -----------------------------------------------
 
```

Comment:
I can't make a comment in the gray sections, but in line 101, it could be changed to do the navigation, then a direction to  'click the new button', since 'new' is added in the menu selection steps, which typically we don't do (at least I didn't think we did).

Reply:
This appears twice in the doc, FYI